### ISSUE TITLE
Link fails on Node 8

### ISSUE THREAD

**Issue by atomicpages** [whitecolor/yalc#85]
I sometimes switch between different node versions and I noticed `yalc link` fails with node 8 today.

```
(node:94815) UnhandledPromiseRejectionWarning: TypeError: Symbol.asyncIterator is not defined.
    at __asyncValues (/Users/USER/.npm/_npx/94815/lib/node_modules/yalc/src/sync-dir.js:38:38)
    at Object.<anonymous> (/Users/USER/.npm/_npx/94815/lib/node_modules/yalc/src/sync-dir.js:99:33)
    at step (/Users/USER/.npm/_npx/94815/lib/node_modules/yalc/src/sync-dir.js:32:23)
    at Object.next (/Users/USER/.npm/_npx/94815/lib/node_modules/yalc/src/sync-dir.js:13:53)
    at fulfilled (/Users/USER/.npm/_npx/94815/lib/node_modules/yalc/src/sync-dir.js:4:58)
    at <anonymous>
(node:94815) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
```

I reckon it's caused by this line here: https://github.com/whitecolor/yalc/blob/master/src/sync-dir.ts#L56.

Adding `ESNext.AsyncIterable` to the `tsconfig` might do the trick 👍

**Comment by whitecolor** [comment]
can you test and make PR for this?

**Comment by atomicpages** [comment]
You bet

**Comment by atomicpages** [comment]
Fixed in PR #86
